models/marts/exposures/openoa/intermediate/int_openoa_plant_data__resampled.sql

```diff
@@ -0,0 +1,41 @@
+{{config(materialized='table')}}
+{% set src_model = 'fct_entr_plant_data' %}
+
+with
+    src as (select * from {{ ref('fct_entr_plant_data') }}),
+    plant_dim as (select * from {{ ref('dim_asset_wind_plant') }}),
+    tag_dim as (select * from {{ ref('dim_entr_tag_list') }}),
+
+    src_filt as (
+        select * from src
+        where entr_tag_id in (2554,2553,2555)
```

Comment:
I'm still unsure whether to do the filtering in these models via tag name or ID, but it feels like the readability of using the tag names is more valuable than the query efficiency here.